tests/commons/glob/test_glob.py

```
@@ -0,0 +1,100 @@
+    assert some_files
+    assert len(some_files) == 3
+
+
+@pytest.mark.parametrize("path, glob, expected", [
+    ("./sub/file.txt", "**/*.txt", True),
+    ("./sub/file.txt", "**/f*.txt", True),
+    ("./sub/file.txt", "**/1*.txt", False),
+    ("./sub/file.txt", "*.txt", True),
+    ("./sub/file.txt", "*.xml", False),
+])
+def test_pathMatchesGlob(path, glob, expected):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_pathMatchesGlob`

Add return type annotation: `None`

(ANN201)